src/System.Security.Cryptography.Cng/src/ApiCompatBaseLine.net463.txt

```diff
@@ -0,0 +1,9 @@
+MembersMustExist : Member 'System.Security.Cryptography.CngAlgorithm.ECDiffieHellman.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
Can you add some comment explaining why these are here and what issue is tracking removing them?